R/calendar_range.R

```diff
@@ -16,12 +16,15 @@
 #' @param end_placeholder Text visible in the end calendar input when nothing is inputted.
 #' @param min Minimum allowed value in both calendars.
 #' @param max Maximum allowed value in both calendars.
+#' @param start_label Label text in the start calendar.
+#' @param end_label Label text in the end calendar.
```

Comment:
How about changing the descriptions to 'Label to be displayed with the start calendar' and 'Label to be displayed with the end calendar'?